#ifndef CREATOR_H
#define CREATOR_H

//ui include
#include "ui_Creator.h"

class Creator: public QMainWindow
{
  private:
    //member variables
    Ui_creator* m_ui;

  public:
    //constructors
    Creator();
    ~Creator();

    //public functions

    //getters

    //setters

  signals:

  public slots:
}; //end class Creator: public QMainWindow

#endif //end ifndef CREATOR_H

